scripts/exgdas_global_marine_analysis_vrfy.py

```diff
@@ -189,3 +196,39 @@ def plot_zonal_slice(config):
 # Sea surface height
 config.update({'variable': 'ave_ssh', 'bounds': [0, 0.1]})
 plot_horizontal_slice(config)
+
+#######################################
+# eva plots
+
+evadir = os.path.join(HOMEgfs, 'sorc', 'gdas.cd', 'ush', 'eva')
+yamlgen = os.path.join(evadir, 'gen_eva_obs_yaml.py')
+marinetemplate = os.path.join(evadir, 'marine_gdas_plots.yaml')
+marinepost = os.path.join(evadir, 'marine_eva_post.py')
+varyaml = os.path.join(comout, 'yaml', 'var.yaml')
+
+# it would be better to refrence the dirs explicitly with the comout path
+# but eva doesn't allow for specifying output directories
+os.chdir(comout)
+if not os.path.exists('preevayamls'):
+    os.makedirs('preevayamls')
+if not os.path.exists('evayamls'):
+    os.makedirs('evayamls')
+
+# mama, i'm sorry
+runlist = [yamlgen, '-i', varyaml,
+                    '-t', marinetemplate,
+                    '-o', 'preevayamls']
+
+subprocess.run(runlist, check=True)
```

Comment:
isn't that py script 'importable'? if so, why not just call the main function within?

Reply:
my bad